### DISCUSSION TITLE
What logs are in a log drain?

### DISCUSSION THREAD

**Discussion by garden-lord** [vercel/community#102] [Show & Tell]
I couldn't find any good docs on what exactly is sent to log drains. I'm primarily interested in traffic but from a simple log drain I have set up I only see events for build logs.https://vercel.com/blog/log-drains seems to imply that traffic should be included, I would guess it's not real time but I am not seeing anything other than build logs even after running the log drain for a couple hours.

**Comment by creamstew** [comment]
The same thing happened to me.

In my case I use datadog as Log Drains.
Build log and Function log are forwarded, but only Access log is not forwarded.
I tried with Sematext, but the same problem happened.

**Comment by pcjmfranken** [comment]
Can confirm missing access log events that up until recently would have been included. I even checked the raw logdrain API output to make sure I wasn't inadvertently filtering them out,

Could this be related to any of the recent (and otherwise awesome 😉) infrastructure work, @steven-tey?

**Reply by steven-tey** [reply]
Hey @pcjmfranken! We basically forward every entry a user can see under "logs" for a specific deployment. If it's not there, then the log drain (and therefore any other integration) won't get that data. May I know which log drain you are using so we can dig into it a little bit more?

**Reply by creamstew** [reply]
Hey @steven-tey! The same thing happened to me. In my case, I tried using Datadog, Logflare and Sematext. The result was the same, only the access log was not forwarded. So I don't think it is caused by a third party logging provider.

**Comment by leerob** [comment]
Our documentation on logging has improved quite a bit since this question was asked, check it out 😄 

https://vercel.com/docs/concepts/deployments/logs

**Comment by creamstew** [comment]
Hey @leerob !
I looked at the documentation, but I couldn't find anything that would solve this problem.
Where can I find a solution?

**Reply by leerob** [reply]
Could you be more clear on your specific problem that wasn't solved by the docs? 🙏
